templates/institutional/institution-landing-fr.html

```diff
@@ -0,0 +1,186 @@
+---
+title: "[Institution-landing-page]"
+layout: no-container
+language: "fr"
+altLangPage: "institution-landing-en.html"
+pageclass: "page-type-nav"
+pageType: "institution-landing"
+breadcrumb: [
+	{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" },
+	{ "title": "Departments & agencies", "link": "https://www.canada.ca/en/government/dept.html" }
```

Comment:
Is this the right breadcrumb that all institutional page use

Reply:
@duboisp The BA has put it as mandatory in their business requirement on WET-153 (see no 2).